deliverables/submitted/requirements.md

```diff
@@ -1,32 +1,242 @@
+1. Response Time: We are not certain on the latency associated with certain functionalities such as the web scrubbing utility.
+1. Reliability
+1. Portability: We are not sure if this piece of software will run on just any operating system.
+1. Accuracy:
+
+It is hard to judge the accuracy of the data mined with our web scraping utility.
+
+5. **MVP**
+
+What will be your MVP? Which features are you going to validate? How?
+
+*Grading criteria* (2 points): Describe what would be considered the Minimal Viable Product and how it will be tested (e.g., via implementation, prototyping, Wizard of Woz, etc.). Make clear what you are going to validate. The MVP should be coherent with the previous sections.
+
+Our Minimum Viable Product should at least be able to properly incorporate our required features for a product category such as our currently planned focus on computer processors. Our webpage will show various processors of different brands and prices, and be able to display these list prices over the past 30 and 90 day periods from at least 4 different sources - namely Amazon, Newegg, Best Buy, and either Walmart or Gamestop with preference towards Gamestop. These will all have valid links to the product page. Used markets will be avoided, although links to ebay search results will be included on the webpage. In addition to price data, pictures from multiple sources will be collected directly from the listings (so as to avoid possible google errors) as well as a general description of the product in addition to relevant specifications. These will be implemented through HTML and PyScript, and utilizing relevant APIs in calculations and data. This website will not have a registered domain name for testing, nor will we seek to establish an accessible server. Instead we will use a tool which allows hosting a local webpage to test our products usage with the APIs. The data should be consistent with Honey price tracking extension and show all these webpages at once with prices over the formerly described intervals. The webpage should be properly formatted and navigable between off-site and on-site links (related products, homepage, and categories of which we will have one). The pictures should also be fully viewable, and description/specs legible on desktop.
+
+6. **Use cases**
+1. **Use case diagram**
+
+Include a UML use case diagram for your project. There are many drawing tools that you can use, such as <https://app.diagrams.net/> and <https://creately.com/>
+
+*Grading criteria* (5 points): Follow correctly the UML specification. The actors should be coherent with what was listed in sections 1 and 2. The use case diagram should be coherent with the list of requirements (section 3). The level of granularity of each use case should be adequate. The use cases should be adequately named.
+
+![](Aspose.Words.2dfcf8d4-4087-4a8f-9f9f-da974c5bf25d.002.png)
+
+2. **Use case descriptions and interface sketch**
```

Comment:
Use the format of dialog between user and system for the description of the use cases